posts/how-to-find-error-messages-with-sourcegraph.md

```diff
@@ -0,0 +1,34 @@
+---
+title: How To Find Error Messages with Sourcegraph (Video)
+author: marek-zaluski
+tags: [video, search, sourcegraph]
+description: Learn about finding error messages in your code with Sourcegraph.
+image: https://i3.ytimg.com/vi/M4zlcZJ2fOA/maxresdefault.jpg
```

Comment:
Could we bring this image into the repo?

Reply:
Sure, sounds good. The only advantage of the direct URL is that it updates automatically if we go into YT and edit the video's thumbnail. I don't expect that we'll be updating thumbnails often so it's not so relevant as an advantage.